src/components/navigation/nav-tab-panel.tsx

```diff
@@ -96,8 +99,8 @@ export class NavTabPanel extends BaseComponent<IProps, IState> {
               })
             }
           </Tabs>
-          {this.state.showChatColumn && focusDocument &&
-            <ChatPanel activeNavTab={activeNavTab} documentKey={focusDocument}
+          {showChatPanel &&
```

Comment:
Similarly, this seems to suggest that unlike the code in lines 81-82, in this case we'll show the chat panel if `showChatPanel` is true even if `isChatEnabled` is false, which is either misleading or wrong. Unless I'm missing something it seems to me that the code can be made clearer and more robust by combining into a single boolean as I did above.